pwa-backend/src/main/java/io/growingabit/app/model/User.java

```diff
@@ -19,10 +20,13 @@
   @Id
   String id;
   @Load
-  List<Ref<SignupStage>> signupStages;
+  Map<String, Ref<SignupStage>> signupStages;
+  @Load
+  Map<String, Ref<SignupStage>> mandatorySignupStages;
 
   public User() {
-    this.signupStages = new LinkedList<>();
```

Comment:
Cambiando il tipo in HashSet perdi l'ordine di inserimento degli stages: non sarebbe meglio usare un LinkedHashSet?

Reply:
Non credo che l'ordine sia importante, tanto poi sono inserite in una mappa `<codice -
 stage>` all'interno del profilo utente, però ok, possiamo usare tranquillamente un LinkedHashSet.